man/systemd.link.xml

```diff
@@ -697,6 +697,27 @@
           <para>Takes a integer. Specifies the NIC transmit ring buffer size. When unset, the kernel's default will be used.</para>
         </listitem>
       </varlistentry>
+      <varlistentry>
+        <term><varname>RxFlowControl=</varname></term>
+        <listitem>
+          <para>Takes a boolean. When set, enables the receive flow control, also known as the receive
+          pause parameter. When unset, the kernel's default will be used.</para>
+        </listitem>
+      </varlistentry>
+      <varlistentry>
+        <term><varname>TxFlowControl=</varname></term>
+        <listitem>
+          <para>Takes a boolean. When set, enables the transmit flow control, also known as the
+          transmit pause parameter. When unset, the kernel's default will be used.</para>
```

Comment:
Takes a boolean. When set, enables the transmit flow control, also known as the ethernet transmit PAUSE message (respond to received Ethernet PAUSE frames).